app/src/main/scala/org/alephium/explorer/persistence/queries/InputQueries.scala

```diff
@@ -37,9 +37,10 @@ object InputQueries extends CustomTypes {
 
   /** Inserts inputs or ignore rows with primary key conflict */
   def insertInputs(inputs: Iterable[InputEntity]): DBActionW[Int] =
-    QuerySplitter.splitUpdates(rows = inputs, columnsPerRow = 9) { (inputs, placeholder) =>
-      val query =
-        s"""
+    if (inputs.nonEmpty) {
+      QuerySplitter.splitUpdates(rows = inputs, columnsPerRow = 9) { (inputs, placeholder) =>
```

Comment:
`QuerySplitter.splitUpdates` already checks for empty inputs [here](https://github.com/alephium/explorer-backend/blob/master/app/src/main/scala/org/alephium/explorer/persistence/queries/QuerySplitter.scala#L123). The [test-case](https://github.com/alephium/explorer-backend/blob/master/app/src/test/scala/org/alephium/explorer/persistence/queries/QuerySplitterSpec.scala#L66). So this check `if (inputs.nonEmpty)` is probably not needed?

Reply:
oh great, I miss that :+1: I will update

Reply:
I just removed the last commit